docs/index.html

```diff
@@ -51,765 +61,903 @@
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "AWS",
+          "label": "gradle",
           "ring": 0,
-          "quadrant": 2,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "aws s3",
+          "label": "Webpack",
           "ring": 0,
-          "quadrant": 3,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "prometheus",
-          "ring": 2,
-          "quadrant": 2,
+          "label": "CodeQL",
+          "ring": 0,
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": -1,
-          "link": ""
+          "moved": 1,
+          "link": "https://codeql.github.com/"
         },
         {
-          "label": "datadog",
+          "label": "Copilot",
           "ring": 0,
-          "quadrant": 2,
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
+
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 0, Languages & Tools
+        // TRIAL
+        /////////////////////////////////////////////////////    
         {
-          "label": "aws athena",
-          "ring": 0,
-          "quadrant": 3,
+          "label": "appsweep",
+          "ring": 2,
+          "quadrant": 0,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://www.guardsquare.com/appsweep-mobile-application-security-testing"
+        },
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 0, Languages & Tools
+        // ASSESS
+        /////////////////////////////////////////////////////
+        {
+          "label": "rust",
+          "ring": 2,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "github actions",
-          "ring": 1,
-          "quadrant": 2,
+          "label": "develocity",
+          "ring": 2,
+          "quadrant": 0,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://gradle.com/develocity/"
+        },{
+          "label": "angular es build",
+          "ring": 2,
+          "quadrant": 0,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://v17.angular.io/guide/esbuild"
+        },
+        {
+          "label": "backstage",
+          "ring": 0,
+          "quadrant": 0,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://backstage.io/"
+        },
+
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 0, Languages & Tools
+        // HOLD
+        /////////////////////////////////////////////////////
+        {
+          "label": "NPM client",
```

Comment:
```suggestion
          "label": "NPM",
```

Reply:
we have npm in adopt, so this is really about the client only. I'm assuming the CLI?
```suggestion
          "label": "npm CLI",
```

Reply:
Wait NPM should only be in adopt. There is no point in holding it, that's what I mentioned in the session. Didn't we agree on that? And then it should only be NPM, or lowercase, I don't care about that.